cypress/support/step_definitions/annual-review/scheduler/todays_date_is_string_days_after_annual_review.js

```diff
@@ -0,0 +1,8 @@
+import { addDays, startOfDay } from "date-fns";
+
+And("todays date is {string} days after annual review", async (date) => {
+  const currentDate = new Date();
+  const futureDate = startOfDay(addDays(currentDate, date));
+
+  await cy.task("worker", {futureDate: futureDate.getTime()});
```

Comment:
is futureDate ever used?

Reply:
Good point. This is something from the previous approach. I'll get rid of it.